net/uspot/Makefile

```diff
@@ -8,9 +8,9 @@ PKG_MAINTAINER:=Thibaut VARÈNE <hacks@slashdirt.org>
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://github.com/f00b4r0/uspot.git
-PKG_SOURCE_DATE:=2024-01-09
-PKG_SOURCE_VERSION:=c4b6f2f0bb1e9d2da4adc8cb3523cd7e440d7584
-PKG_MIRROR_HASH:=fa6be10e0479a9dc71b8c21e57b07aac09c8938e8e7120045816a5cd4b949343
+PKG_SOURCE_DATE:=2024-03-25
```

Comment:
I believe this needs to be changed from - to . because of apk.

Reply:
Well I used `update_git_source_package.sh` from up-to-date maintainer-tools repo so if the output is wrong, the script must be updated.

That said, I sure hope that old style dates aren't breaking anything otherwise there will be a transition period where existing packages will be uninstallable, among other things.

Reply:
Also this comment seems to suggest nothing should be done for now: https://github.com/openwrt/packages/issues/23706#issuecomment-2016507940 ?